Detectors/TPC/reconstruction/macro/findKrBoxCluster.C

```diff
@@ -58,44 +58,28 @@ void findKrBoxCluster(int lastTimeBin = 1000, int run = -1, int time = -1, std::
     clFinder->loadGainMapFromFile(gainMapFile);
   }
 

```

Comment:
I propose to remove
```c++
auto clFinder = std::make_unique<o2::tpc::KrBoxClusterFinder>();
```
here, and the above, when the tree is created use replace
```c++
std::vector<o2::tpc::KrCluster> clusters;
```
by
```c++
auto clFinder = std::make_unique<o2::tpc::KrBoxClusterFinder>();
auto& clusters = clFinder->getClusters();
```
This will avoid the unnecessary copy of the clusters.
See also comment below.